contents/blog/the-posthog-array-1-30-0.md

```diff
@@ -0,0 +1,130 @@
+---
+date: 2021-11-17
+title: Array 1.30.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes"]
+featuredImage: ../images/blog/array/1-28-0.png
+excerpt: Excerpt here
+---
+
+
+PostHog 1.30.0 includes a description here.
+
+## PostHog 1.30.0 release notes
+
+> For a better experience and new features, remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Fresh new look-and-feel](#fresh-new-look-and-feel)
+- [Correlation analysis](#correlation-analysis)
+- [Saved insights](#saved-insights)
+- [Fully revamped recordings](#fully-revamped-recordings)
+
+### Fresh new look-and-feel
+
+PostHog just got a lot fresher! We have a brand new UI and layout that had been on the works, codename `lemonade` (because it's fresh). It's not only a coat of paint, we've pushed a lot of improvements to overall navigation and performance, we call it turbo mode.
+
+
+{images here}
```

Comment:
@clarkus I know you've put together some images recently (I believe for the launch), can you share the images we should add for this array pls?

Reply:
@Twixes if you want to review this (or the entire thing too 😉)

Reply:
How's this @paolodamico? I have many more options at https://www.figma.com/file/NHPA2hVWwaKp4eTGWdigs9/?node-id=0%3A1 

![schenanigans header tan](https://user-images.githubusercontent.com/254612/141598518-fe832b04-4e95-4899-9911-7c0a0a3e60a3.png)

Reply:
I would strongly prefer screenshots over mockups here. I can take some showing off the UI refresh.

Reply:
That makes sense to me @Twixes! I'll ping you on Slack to get some cool screenshots

Reply:
Alright, here's a composition showing off a few aspects of the new UI:
<img width="1527" alt="lemonade" src="https://user-images.githubusercontent.com/4550621/142001323-aa0f4d8c-229f-488a-9e7e-89a24c8c5762.png">

Reply:
Oops, the mobile screenshot was a bit wrong, now correct:
<img width="1527" alt="Lemon Composition" src="https://user-images.githubusercontent.com/4550621/142002267-7f91887f-4415-4445-bbad-6dfeb65be993.png">